products/bash/release-mq.sh

```diff
@@ -17,7 +17,7 @@
 #   -r : <release_name> (string), Defaults to "mq-demo"
 #   -i : <image_name> (string)
 #   -q : <qm_name> (string), Defaults to "QUICKSTART"
-#   -t : optional flag to enable tracing
```

Comment:
How come we've removed the docs for -t, but left it in as a param?

Reply:
missed it. added it now